src/django/api/extended_fields.py

```diff
@@ -16,23 +16,54 @@ def extract_range_value(value):
 MAX_PRODUCT_TYPE_COUNT = 50
 
 
+def get_facility_and_processing_type_extendfield_value(field, field_value):
```

Comment:
No logic was changed in this function and the one below. They were just pulled out of the `create_extendedfield` function so they could be reused by `create_extendedfields_for_claim`